lib/internal/url.js

```diff
@@ -219,6 +219,14 @@ class URL {
     if (base !== undefined && !(base instanceof URL))
       base = new URL(String(base));
     parse(this, input, base);
+
+    Object.defineProperty(this, Symbol.toStringTag, {
+      value: 'URL',
+      writable: false,
+      enumerable: false,
+      configurable: true
+    });
```

Comment:
Web IDL requires that. The class string of the prototype should be `URLPrototype`.